### PR TITLE
enh: improve copy in assistant builder slack integration

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: front/components/assistant_builder/AssistantBuilder.tsx [hunk 1]
```diff
@@ -1221,7 +1222,7 @@ function SlackIntegration({
       {existingSelection.length ? (
         <>
           <div className="mt-6 text-sm text-element-700">
-            Your assistant will answer by default when @dust is mentioned in the
+            Your assistant will answer by default when @Dust is mentioned in the
```
Comment (PopDaph) [inline front/components/assistant_builder/AssistantBuilder.tsx:1225]: NIT NIT NIT: I would say "This assistant will answer" or "@name_of_the_assistant will answer"

**Review by PopDaph — DISMISSED** [review]

File: front/components/assistant_builder/AssistantBuilder.tsx (hunk 1 above)
Comment (fontanierh) [reply front/components/assistant_builder/AssistantBuilder.tsx:1225]: agreed that's better 👍

Comment (Duncid) [reply front/components/assistant_builder/AssistantBuilder.tsx:1225]: agreed
@dust not @Dust

Comment (fontanierh) [reply front/components/assistant_builder/AssistantBuilder.tsx]: Updated + added some other small improvements

Comment (fontanierh) [reply front/components/assistant_builder/AssistantBuilder.tsx]: > @dust not @Dust

@Duncid  on slack it is `@Dust`

Screenshot 2023-10-10 at 17 11 47

**Pull request merged**